src/components/Buttons/IconButton/IconButton.scss

```diff
@@ -0,0 +1,32 @@
+@import '../Button/Button';
+
+.IconButton {
+    &--size {
+        &_small {
+            padding: var(--space-75);
+        }
+        &_medium {
+            padding: var(--space-100);
+        }
+        &_large {
+            padding: 10px;
```

Comment:
Please double check with Dima. We shouldn't use anything that isn't in the tokens.